android/src/main/java/com/instabug/flutter/modules/BugReportingApi.java

```diff
@@ -0,0 +1,166 @@
+    public void show(@NonNull String reportType, @Nullable List<String> invocationOptions) {
+        int[] options = new int[invocationOptions.size()];
+        for (int i = 0; i < invocationOptions.size(); i++) {
+            options[i] = ArgsRegistry.getDeserializedValue(invocationOptions.get(i));
+        }
+        int reportTypeInt = ArgsRegistry.getDeserializedValue(reportType);
+        BugReporting.show(reportTypeInt, options);
+    }
+
+    @Override
+    public void setInvocationEvents(@NonNull List<String> events) {
+        new Handler(Looper.getMainLooper()).post(new Runnable() {
+            @Override
+            public void run() {
+                InstabugInvocationEvent[] invocationEventsArray = new InstabugInvocationEvent[events.size()];
+                for (int i = 0; i < events.size(); i++) {
+                    String key = events.get(i);
+                    invocationEventsArray[i] = ArgsRegistry.getDeserializedValue(key);
+                }
+                BugReporting.setInvocationEvents(invocationEventsArray);
+            }
+        });
+    }
+
+    @Override
+    public void setReportTypes(@NonNull List<String> types) {
+        new Handler(Looper.getMainLooper()).post(new Runnable() {
+            @Override
+            public void run() {
+                int[] reportTypesArray = new int[types.size()];
+                for (int i = 0; i < types.size(); i++) {
+                    String key = types.get(i);
+                    reportTypesArray[i] = ArgsRegistry.getDeserializedValue(key);
+                }
+                BugReporting.setReportTypes(reportTypesArray);
+            }
+        });
+    }
+
+    @Override
+    public void setExtendedBugReportMode(@NonNull String mode) {
+        final ExtendedBugReport.State resolvedMode = ArgsRegistry.getDeserializedValue(mode);
+        BugReporting.setExtendedBugReportState(resolvedMode);
+    }
+
+    @Override
+    public void setInvocationOptions(@NonNull List<String> options) {
+        int[] resolvedOptions = new int[options.size()];
+        for (int i = 0; i < options.size(); i++) {
+            resolvedOptions[i] = ArgsRegistry.getDeserializedValue(options.get(i));
+        }
+        BugReporting.setOptions(resolvedOptions);
+    }
+
+    @Override
+    public void setFloatingButtonEdge(@NonNull String edge, @NonNull Long offset) {
+        final InstabugFloatingButtonEdge resolvedEdge = ArgsRegistry.getDeserializedValue(edge);
+        BugReporting.setFloatingButtonEdge(resolvedEdge);
+        BugReporting.setFloatingButtonOffset(offset.intValue());
+    }
+
+    @Override
+    public void setVideoRecordingFloatingButtonPosition(@NonNull String position) {
+        final InstabugVideoRecordingButtonPosition resolvedPosition = ArgsRegistry.getDeserializedValue(position);
+        BugReporting.setVideoRecordingFloatingButtonPosition(resolvedPosition);
+    }
+
+    @Override
+    public void setShakingThresholdForiPhone(@NonNull Double threshold) {
```

Comment:
None I am aware of. I don't think they handle such use case since it's not very common.
I expect platform-specific APIs to be merged in the future anyway.